src/main/java/net/countercraft/movecraft/warfare/MovecraftWarfare.java

```diff
@@ -82,8 +81,32 @@ public void onEnable() {
             Config.AssaultOwnerWeightPercent = getConfig().getDouble("AssaultOwnerWeightPercent", 1.0);
             Config.AssaultMemberWeightPercent = getConfig().getDouble("AssaultMemberWeightPercent", 1.0);
             Config.AssaultDestroyableBlocks = new HashSet<>();
-            for(String s : getConfig().getStringList("AssaultDestroyableBlocks")) {
-                Config.AssaultDestroyableBlocks.add(Material.getMaterial(s));
+
+            boolean usingBlockIds = false;
+
+            for(String materialInputString : getConfig().getStringList("AssaultDestroyableBlocks")) {
+                Material material;
+
+                try {
+                    material = Material.getMaterial(Integer.parseInt(materialInputString));
+                    usingBlockIds = true;
+                } catch (NumberFormatException exception) {
+                    material = Material.getMaterial(materialInputString.toUpperCase());
+                }
+
+                if (material == null) {
+                    getLogger().severe("Invalid AssaultDestroyableBlock: " + materialInputString);
+                } else {
+                    Config.AssaultDestroyableBlocks.add(material);
+                }
+            }
+
+            if (usingBlockIds) {
+                getLogger().severe("+-------------------------------------------------------------------+");
+                getLogger().severe("|           You are using Block IDs in your configuration.          |");
+                getLogger().severe("|      Please update your configuration to use material names:      |");
+                getLogger().severe("| https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Material.html |");
+                getLogger().severe("+-------------------------------------------------------------------+");
```

Comment:
Let's try to keep PRs to a single change.

Reply:
I'll move it into its own PR, I wasn't sure if you wanted all of #10 moved into this PR.

Reply:
My apologies on that, I forgot eirik had this in his PR.  This for some reason broke assaults.

At sometime in the near future (probably before v0.6) I'll be forcing server owners to use material names rather than IDs (similar to how Movecraft-Combat does it).

Reply:
I checked his code when porting this over, his broke assaults because his code was written to check if the string grabbed from the config was an instance of an integer.

Inherently, strings are not integers, therefore it resulted in no destroyable blocks being loaded.

The code above should function properly; it's untested as I have no in-game experience with assaults.

I can commit it to this PR again if you'd like as a first step, I can PR it separately, or I can leave it out entirely and you can add in your solution that forces material names.